create_jenkins_job.py

```diff
@@ -183,6 +184,15 @@ def create_job(os_name, job_name, template_file, additional_dict):
             'mailer_recipients': DEFAULT_MAIL_RECIPIENTS,
         })
 
+        # configure nightly job for compiling with clang on linux
+        if os_name == 'linux':
+            create_job(os_name, 'nightly_' + os_name + '_clang', 'ci_job.xml.em', {
+                'cmake_build_type': 'Debug',
```

Comment:
Is there a reason for the Debug build type being used as the default here? The default build type for CI jobs is usually `None`.

Reply:
All nightly jobs use a specific build type. Only the manual triggered ones don't choose one by default. So using `Debug` seems fine - not sure if also adding `Release` would catch more cases.